OWML.ModHelper.Menus/ModButton.cs

```diff
@@ -22,6 +23,17 @@ public int Index
                 Button.transform.SetSiblingIndex(value);
             }
         }
+        public bool IsSelected => _uIStyleApplier.GetValue<bool>("_selected");
```

Comment:
can _uIStyleApplier be null? If so, this'll probably do:
`_uIStyleApplier?.GetValue<bool>("_selected") ?? false;`

Reply:
ah, didn't think to use `??`

Reply:
Wait for sometime though I want to implement ModPromptButton in a separate branch first

Reply:
done